src/CMakeLists.txt

```diff
@@ -24,6 +24,7 @@ include_directories(${PYBIND11_INCLUDE_DIR} ${PYTHON_INCLUDE_DIRS})
 # Add executables
 # =============================================================================
 add_executable(nmodl main.cpp)
+set_target_properties(nmodl PROPERTIES PUBLIC_HEADER nmodl.hpp)
```

Comment:
I see it used to get installed and I like one change at a time, so no need for changes. However, did you happen to find out why we need this? To my knowledge NMODL is an application (not library) and a Python module. Hence, it's not obvious to me why we need this.

Reply:
Indeed, it seems to be only used by the unit tests. I can try to remove it, together with adding some "legacy" glue.

Reply:
No legacy glue please.